WindowsServerDocs/storage/file-server/smb-over-quic.md

```diff
@@ -11,7 +11,7 @@ ms.localizationpriority: medium
 
 # SMB over QUIC (PREVIEW)
 
->Applies to: Windows Server 2022 Datacenter: Azure Edition Preview, Windows 11 Insider Preview 
+>Applies to: Windows Server 2022 Datacenter Azure Edition Preview, Windows 11 Insider Preview 
```

Comment:
Please revert, this is not correct. The name of the OS contains a colon, the name returned INSIDE the OS via API will contain no colon due to it causing application incompatibilities.

Reply:
Oops, you're right! My mistake!